apps/rewards/app/components/Panel/NewReward/TokenSelectorInstance.js

```diff
@@ -40,7 +40,6 @@ const IconSpacer = styled.div`
 `
 
 const TokenName = styled.span`
-  max-width: 110px;
```

Comment:
I guess if `max-width` was used, it was a mechanism to control the size of the element, we need to check that wider elements don't break the container box

Reply:
I tested that by using a very long token name, and it wraps at the size of the container (minus margin), which is what we want. I still have no idea what was the intent of using that here, but as far as I'm concerned, it's simply not needed.